pennylane/interfaces/batch/autograd.py

```diff
@@ -113,13 +113,13 @@ def _execute(
             # For backwards compatibility, we flatten ragged tape outputs
             # when there is no sampling
             r = np.hstack(res[i]) if res[i].dtype == np.dtype("object") else res[i]
-            res[i] = np.tensor(r)
+            res[i] = np.tensor(r, requires_grad=True)
 
         elif isinstance(res[i], tuple):
-            res[i] = tuple(np.tensor(r) for r in res[i])
+            res[i] = tuple(np.tensor(r, requires_grad=True) for r in res[i])
 
         else:
-            res[i] = qml.math.toarray(res[i])
+            res[i] = qml.math.toarray(res[i], requires_grad=True)

```

Comment:
Here is another change I made; I noticed that the autograd interface was now always returning tensors with `requires_grad=False`, which meant Autograd was refusing to differentiate the output of QNodes! Here I did a brute force change, _but_ I wonder if we should consider some nuance here.